assets/app/views/_template.html

```diff
@@ -0,0 +1,7 @@
+<li class="template">
+  <img ng-src="{{template.icon}}">
+  <label class="type-label">{{template.category}}</label>
```

Comment:
is the label directive something we get for free? not seeing where its been defined